src/Microsoft.DotNet.Interactive.VSCode/KernelExtensions.cs

```diff
@@ -8,13 +8,13 @@ namespace Microsoft.DotNet.Interactive.VSCode
 {
     public static class KernelExtensions
     {
-        public static Task UseVSCodeHelpersAsync<TKernel>(this TKernel kernel, Kernel rootKernel) where TKernel : DotNetKernel
+        public static Task UseVSCodeHelpersAsync<TKernel>(this TKernel kernel, Kernel rootKernel) where TKernel : ISupportSetValue
```

Comment:
We should take the same approach as #1489.

Reply:
that is not done and not passing